src/main/java/frc/robot/subsystems/color_wheel/commands/Spin.java

```diff
@@ -0,0 +1,69 @@
+package frc.robot.subsystems.color_wheel.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.color_wheel.ColorWheel;
+
+import java.util.ConcurrentModificationException;
+
+/**
+ * Spin the wheel 3 times.
+ */
+public class Spin extends CommandBase {

```

Comment:
Try to find an easier way to create this command

Reply:
How?

Reply:
You can do it by checking only if the same color has been seen, not for every color, you can compare the color before updating and after in order to check if the color has changed, or remained the same